src/components/Market.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react'
+import { inject, observer } from 'mobx-react'
+import Item from './Item'
+
+const Market = inject("Inventory")(observer((props) => {
+    const [newItem, setNewItem] = useState({ name: '', quantity: 0, price: 0 })
+
+    const handleInput = ({ target }) => {
```

Comment:
This function can just be:

`const handleInput = ({ target }) => setNewItem({ ...newItem, [target.name]: target.value })`